src/solvers/refinement/string_refinement.cpp

```diff
@@ -654,7 +657,7 @@ decision_proceduret::resultt string_refinementt::dec_solve()
     {
       bool satisfied;
       std::vector<exprt> counter_examples;
-      std::tie(satisfied, counter_examples)=check_axioms(
+      std::tie(satisfied, counter_examples) = check_axioms(
```

Comment:
Whitespace change?

Reply:
Function arguments change triggered reformatting of this line.

Reply:
I'm failing to see a difference between those two lines except for the whitespace around `=`. Would you mind guiding me?

Reply:
Lines 669 and 670